### ISSUE TITLE
Removing inline function parenthesis, itself removing right side parenthesis also

### ISSUE THREAD

**Issue by kevinemk** [adobe/brackets#13399]
I am using brackets for editing java script. Lets say we have to log some function's value in console. We use something like this. `console.log(myFunctionName());` .But when I remove function right and left parenthesis by pressing back button from the key board again itself it removed right parenthesis of the console.log function. `console.log(myFunctionName;`

### Steps to Reproduce

1. set cursor between myFunctionName() end of console log parenthesis like - > console.log(myFunctionName()|) 
2. Press back key two times to remove the parenthesis.

**Expected behavior:** Remote last three (3) parenthesis 

**Actual behavior:** no need to remove console.log end parenthesis

### Versions

OS Windows - 10 (64bit)
![bracket](https://cloud.githubusercontent.com/assets/8066214/26574089/b201c49e-453d-11e7-8e92-220cc7d4ad62.gif)

Version of Brackets - Release 1.9 build 1.9.0-17312 (release 189f6d39a)

**Comment by Badestrand** [comment]
>> Expected behavior: Remote last three (3) parenthesis

Wouldn't the expected behaviour be that only two paranthesis behind `myFunctionName` get removed?

**Comment by kevinemk** [comment]
Yes. Just need to remove the `myFunctionName` parenthesis but it removes console.log right side parenthesis also.
